.changeset/metal-cougars-develop.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/checkbox': major
```

Comment:
why would this be major?

Reply:
React guidelines recommend wrapping a component in forwardRef is a major change